include/rsp.inc

```diff
@@ -456,6 +466,28 @@ makeStoreInstruction swv, 0b00111
     .long (0x12 << 26 | 0x0 << 21 | \vreg << 11 | hex.\reg << 16 | \element << 7)
 .endm
 
+# Vector element macros
+#define VE_v  0
+#define VE_0q 2
+#define VE_1q 3
+#define VE_0h 4
+#define VE_1h 5
+#define VE_2h 6
+#define VE_3h 7
+#define VE_0  8
+#define VE_1  9
+#define VE_2  10
+#define VE_3  11
+#define VE_4  12
+#define VE_5  13
+#define VE_6  14
+#define VE_7  15
+
+# Instead of using the constants above, it is possible to use the syntax 'e(x)'
+# via the following macros.
+#define _PPCAT2(n,x) n ## x
+#define _PPCAT(n,x) _PPCAT2(n,x)
+#define e(xx) _PPCAT(VE_, xx)
```

Comment:
Nice! The only issue here is that, looking at rsp_mixer, I don't specifically like `e(v)` much. I don't think it's very clear, it looks more like line noise to me. Maybe we can keep a `,0` in those cases? What do you think?